model/structures.go

```diff
@@ -22,8 +22,8 @@ import (
 // PdfRectangle is a definition of a rectangle.
 type PdfRectangle struct {
 	Llx float64 // Lower left corner (ll).
-	Lly float64
```

Comment:
It's not necessary. I made it because the next text extraction is mostly comparing text bounding boxes and the literals are easier to read as x span, y span. I spent a large fraction of my development time reading bounding box rectangles and this helped me a lot.
Go struct literals without field names are dangerous because they make it easy to get the fields wrong. VS-Code flags them for me.